src/pkg/cli/upgrade.go

```diff
@@ -24,7 +25,7 @@ func Upgrade(ctx context.Context) error {
 	}
 	term.Debugf(" - Evaluated: %s\n", ex)
 
-	if strings.HasPrefix(ex, homebrewPrefix(ctx)) {
+	if prefix, err := homebrewPrefix(ctx); err == nil && strings.HasPrefix(ex, prefix) {
```

Comment:
this is a lot for one line imo
```suggestion
	prefix, err := homebrewPrefix(ctx)
	if err == nil && strings.HasPrefix(ex, prefix) {
```

Reply:
True. I did it like that to limit the scope of the variables.